Tests/Schema.NET.Test/Schema.NET.Test.csproj

```diff
@@ -1,13 +1,27 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup Label="Build">
-    <TargetFrameworks>net7.0;net6.0;net472</TargetFrameworks>
+    <TargetFrameworks>net8.0;net6.0;net462;net472;net48</TargetFrameworks>
```

Comment:
Not sure how necessary it is to test 3 different versions of .NET Framework. I'd personally be happy to just test one .NET Framework version (likely 4.8).

Reply:
Please let me know how to proceed here. Keep all net versions or leave net48 only?

Reply:
Removed redundant versions of older .NET framework